validate.lic

```diff
@@ -331,6 +331,13 @@ class DRYamlValidator
     warn('The analyze_combo_type setting is deprecated and can be removed from your YAML. The use_barb_combos setting should be used instead.')
   end
 
+  def assert_that_prebuff_has_waggle_set(settings)
+    return unless settings.pre_hunt_buff
+    return unless settings.waggle_sets.nil?
```

Comment:
Actually, if we don't use the boolean to determine if we're going to prebuff, then what is there to validate? If we set a prehunt waggle set, that means we're going to be using it to buff before a hunt. What do we check here?